### PR TITLE
Fix Encoding Type in Fireeye Importer

### PR DESCRIPTION
**Pull request by harsh098**
This PR fixes issue #1403

### CODE REVIEW

File: vulnerabilities/importers/fireeye.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def advisory_data(self) -> Iterable[AdvisoryData]:
                 if Path(file).stem == "README":
                     continue
                 try:
-                    with open(file) as f:
+                    with open(file, encoding="utf-8-sig") as f:
```
Comment (TG1999) [inline vulnerabilities/importers/fireeye.py:47]: @harsh098 thanks for your PR, please add tests

**Review by TG1999 — CHANGES_REQUESTED** [review]
Please add tests for verifying the code change :)

File: vulnerabilities/importers/fireeye.py (hunk 1 above)
Comment (harsh098) [reply vulnerabilities/importers/fireeye.py:47]: ok @TG1999

Comment (harsh098) [reply vulnerabilities/importers/fireeye.py:47]: Should I add this test for `md_list_to_dict` or `parse_advisory`.
I am currently going for `md_list_to_dict` as the goal to accomplish is to check if
`md_list_to_dict` works with the `utf-8-sig` encoding or not

**Review by TG1999 — APPROVED** [review]
LGTM!

**Pull request merged**